stacker/providers/aws/default.py

```diff
@@ -1,19 +1,24 @@
 import json
 import logging
 import os
-import time
-
-import botocore.exceptions
 
+import botocore
 from ..base import BaseProvider
 from ... import exceptions
 from ...util import retry_with_backoff
 from stacker.session_cache import get_session
+from cloudsns.cloudlistener import CloudListener
+from stacker.status import (
+    SubmittedStatus,
+    CompleteStatus,
+)
 
 logger = logging.getLogger(__name__)
 
 MAX_TAIL_RETRIES = 5
 
+LISTENER_NAME = "StackerSNSListener"
```

Comment:
This ends up being the queue name, right?  I don't think this should be hard-coded - I think we need to do a review of cloudsns at this point, it was sort of developed with very few PRs.